src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/WaysWeSell.tsx

```diff
@@ -95,34 +98,61 @@ const ShelfItem: React.FC<StepsWithImageBlackDataType> = ({
   title,
   text,
   srcSet,
-}) => (
-  <>
-    <Box maxWidth="100%" bg="black10" mb={[1, 2]}>
-      <Image
-        src={src}
-        srcSet={srcSet}
-        width="100%"
-        height="100%"
-        lazyLoad
-        alt={`${title} image`}
-        style={{
-          display: "block",
-        }}
-      />
-    </Box>
-    {title && (
-      <Text mb={[0.5, 1]} variant={["md", "lg-display", "xl"]} color="white100">
-        {title}
-      </Text>
-    )}
-
-    {text && (
-      <Text variant={["xs", "sm"]} color="white100">
-        {text}
-      </Text>
-    )}
-  </>
-)
+}) => {
+  const { width } = useWindowSize()
+
+  const aspectRatio = BIG_IMAGE_HEIGHT / IMAGE_WIDTH
+
+  // calculating the width of the image contailer
+  // (screen width - HorizontalPadding - px of the parent - gridColumnGap) / 3
+  const collumnHeightMediumcreen =
+    ((width - 20 * 2 - 20 * 2 - 20 * 2) / 3) * aspectRatio
```

Comment:
can we use space(2) here instead of `20`

Reply:
It's deprecated. Are we supposed to use some other hook? I couldn't find any on Force